cmd/kubeadm/app/cmd/config_test.go

```diff
@@ -101,6 +101,10 @@ func TestImagesListRunWithCustomConfigPath(t *testing.T) {
 
 			i, err := cmd.NewImagesList(configFilePath, &kubeadmapiv1alpha1.MasterConfiguration{})
 			if err != nil {
+				// short-circuits for to-be-released new version when bumping in advance
```

Comment:
Can you instead set `kubernetesVersion` specifically on all unit test cases? e.g. lock that to v1.10.3 or v1.11.0.
I think that also should do

Reply:
version label is the dynamic content. for tests it is better to have some static version anyway, as this will make tests deterministic and not dependant on some external infra.

Reply:
Right. Already updated. PTAL. Thanks.